source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
+  display the results of your Playground.
+
+  - If the results can be displayed in |json| format, |vsce| labels the
+    right-side pane **Playground Results.json**.
+
+  - If the results can't be displayed in |json| format, |vsce| labels
+    the right-side pane **Playground Result**.
+
+- If you disabled split-view, |vsce| displays the Playground results in
+  a new tab to the right of your Playground. These results may be in in
```

Comment:
> These results may be in in |json| format if possible.

I wouldn't add this sentence because it is confusing and makes people think that because of the position of the window the results can behave differently. It is not true.